packages/mookme/src/prompts/init.ts

```diff
@@ -12,16 +12,23 @@ export const choiceQuestion = (name: string, message: string, choices: string[])
   pageSize: process.stdout.rows / 2,
 });
 
-export async function selectHookTypes(skip = false): Promise<HookType[]> {
+export async function selectHookTypes(skip = false, typeToHook?: HookType): Promise<HookType[]> {
```

Comment:
Could you add a `todo` here stating:

```
@todo(lecanu.maxence@gmail.com): standardize arguments of this function as a unique argument being an array of types
```

I'll try to go over this technical debt shortly

Reply:
Sure thing 👍